user-guide/Advanced_Functionality/Databases/Elasticsearch_database/Installing_Elasticsearch_on_Linux.md

```diff
@@ -0,0 +1,116 @@
+
+1. Make sure the firewall ports are open for Elasticsearch. Elasticsearch operates on TCP port 9200 and TCP port 9300.
+
+   - There is a default firewall on Linux, but this is disabled by default. To enable the firewall, use the following command:
+
+     `$ systemctl start firewalld.service`
+
+   - To add the correct ports to the firewall, you can for example use the following commands:
+
+     - `$ firewall-cmd --add-port=9200/tcp --permanent`
+
+     - `$ firewall-cmd --add-port=9300/tcp --permanent`
+
+     - `$ firewall-cmd --reload`
+
+   > [!IMPORTANT]
+   >
+   > - These commands could be **different depending on the Linux distribution** you are using. Refer to [the official Elastic page on installing Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html) for detailed information.
+   > - If you connect to your Linux server with **SSH**, you must **immediately exclude port 22**, or you will be locked out of the session. Use the following command for this: `$ firewall-cmd --zone=public --add-port=22/tcp`
+
+1. Mount the data folder to the data disk.
+
+   > [!NOTE]
+   >
+   > - The folder where the Elasticsearch data is stored is configured in *elasticsearch.yaml*, in the property *path.data*.
+   > - To verify on which disk the data is mounted, execute the "df" command.
+
+1. Configure the user rights for the Elasticsearch user/group in the data folder.
+
+   You can do this with the following command:
+
+   `$ chown -R elasticsearch:elasticsearch /directoryname_from_path.data`
+
+1. Configure the *elasticsearch.yml* files.
+
+   - Make sure the Elasticsearch service is stopped. To stop the Elasticsearch service, use the following command:
+
+     `$ sudo systemctl stop elasticsearch.service`
+
+   - To change *elasticsearch.yml* files, you can use the following command:
+
+     `$ vi /etc/elasticsearch/elasticsearch.yml`
+
+   - Make the following adjustments in the *elasticsearch.yml* file based on your setup:
+
+     - **cluster.name**: This needs to be the same for all nodes in your Elasticsearch cluster.
+
+     - **node.name**: The name of the Elasticsearch node.
+
+     - **path.data**: The location(s) where you want to store the data.
+
+     - **bootstrap.memory_lock**: Set this to *true*.
+
+     - **network.host**: The IP address of the node.
+
+     - **http.port**: Set this to *9200*.
+
+     - **discovery.zen.ping.unicast.hosts**: The IP addresses of all the nodes in your Elasticsearch cluster. Elasticsearch nodes use this list of hosts to find each other and learn the topology of the ring.
+
+       > [!NOTE]
+       >
+       > - We recommend a cluster of three nodes, preferably in different racks.
+       > - To add a node to an existing cluster, see [Adding an Elasticsearch cluster node](xref:Configuring_Elasticsearch_node_add).
```

Comment:
Is a very specific setup. I don't believe we recommend a cluster of precisely three nodes, we recommend a cluster of minimum 3 nodes. (because of the master node configuration)